Date/Date.cpp

```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include "Date.h"
+
+using namespace std;
+
+void Date::validate(int day, int month, int year) {
+    int daysInMonth[13] = {1, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
+
+    if ( day > daysInMonth[month] || day < daysInMonth[0]) {
```

Comment:
`day < daysInMonth[0]` in this case is not very obvious.` day < 1` would be better